haskell.el

```diff
@@ -354,8 +354,11 @@ If `haskell-process-load-or-reload-prompt' is nil, accept `default'."
 (defun haskell-mode-after-save-handler ()
   "Function that will be called after buffer's saving."
   (when haskell-tags-on-save
-    (ignore-errors (when (and (boundp 'haskell-session) haskell-session)
-                     (haskell-process-generate-tags))))
+    (ignore-errors
+      (if (and (boundp 'haskell-session)
```

Comment:
Well, yes, at least for now, look at this
https://github.com/haskell/haskell-mode/blob/master/haskell.el#L351